src/neofs_testlib/cli/neogo/contract.py

```diff
@@ -5,9 +5,9 @@
 
 
 class NeoGoContract(CliCommand):
-    def compile(
+    def contract_compile(
         self,
-        input_file: str,
+        i: str,
```

Comment:
Doesn't match the description below and I'd expect `i` to be used for different purpose.

Reply:
I can't replace ``` i ``` with anything else =(
here are options for this command:

```
E           OPTIONS:
E              --in value, -i value        Input file for the smart contract to be compiled
E              --out value, -o value       Output of the compiled contract
E              --verbose, -v               Print out additional information after a compiling
E              --debug value, -d value     Emit debug info in a separate file
E              --manifest value, -m value  Emit contract manifest (*.manifest.json) file into separate file using configuration input file (*.yml)
E              --config value, -c value    Configuration input file (*.yml)
E              --no-standards              do not check compliance with supported standards
E              --no-events                 do not check emitted events with the manifest
E              --no-permissions            do not check if invoked contracts are allowed in manifest
E              --bindings value            output file for smart-contract bindings configuration
```

So i have either ```in``` or ```i```, but former is the reserved python keyword, so I can use only ```i```

Reply:
description updated

Reply:
I've thought it's just an internal name, but it's mapped directly onto CLI parameters, then yes.

Reply:
Do I need to do smth else here?